my-app/src/components/board.js

```diff
@@ -1,11 +1,15 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import Square from './square.js';
+import { GameContext } from './game.js';
 
-const Board = ({ squares, onClick }) => (
-  <div className="board">
-    {squares.map((square, i) => (
-      <Square key={i} value={square} onClick={() => onClick(i)} />
-    ))}
-  </div>
-);
-  export default Board;
+const Board = () => {
+  const { current, handleClick } = useContext(GameContext);
```

Comment:
Add few custom hooks. You can look for example: https://kentcdodds.com/blog/how-to-use-react-context-effectively

Reply:
Why did you remove all your `useContext`'s?  We want to use global state management, there wouldn't be any `useState` or `useReducer` in other places than `GameProvider`.

Reply:
In general, you did great on implementation. I'll only add few minor change request from this point.